proto/trip.proto

```diff
@@ -180,6 +180,7 @@ message TripLeg {
     repeated StreetName tunnel_name = 55;
     float elevation_sampling_interval = 56;
     repeated float elevation = 57;
+    bool ctry_crossing = 58;
```

Comment:
just one nitpick, can you please spell it out? none of the other ones are abbreviated

Reply:
Done